### DISCUSSION TITLE
Error during opening lab Courses/2414/labs/7088

### DISCUSSION THREAD

**Discussion by Liaqath3ra** [ibm-skills-network/author-workbench-knowledgebase#45] [Help]
Whenever I launched the App. White Page appeared on page below mentioned error can anyone guide me how can I resolve this issue.

500

Internal Server Error
Sorry, our servers encountered an unexpected error.

[Go back home](https://author.skills.network/)
[Contact support →](https://github.com/ibm-skills-network/author-workbench-knowledgebase/discussions/new?category=help)

**Comment by michaelroudnitski** [comment]
Hi, this should now be fixed
